## 11_fix_privacy_doc by Jan Wagner
##
## DP: Fixing potential privacy breach by fetching data from sourceforge.net

commit 084af86fe6ee55135144ac44f661ff139434fdc3
Author: Jan Wagner <waja@cyconet.org>
Date:   Sun Mar 2 21:22:42 2014 +0100

    Fix potential privacy breach of HTML documentation

diff --git a/doc/faq.html b/doc/faq.html
index 69c3a97..b421495 100755
--- a/doc/faq.html
+++ b/doc/faq.html
@@ -225,7 +225,7 @@
 	  <td>
              <p class="SectionBody">This project is also hosted by : <br>
 
-             <A href="http://sourceforge.net/projects/nagios-snmp/"> <IMG src="http://sourceforge.net/sflogo.php?group_id=134917&type=5" width="210" height="62" border="0" alt="SourceForge.net Logo" /></A>
+             <A href="http://sourceforge.net/projects/nagios-snmp/">http://sourceforge.net/projects/nagios-snmp</A>
 	  </p></td>
 	</tr>
 
diff --git a/doc/index.html b/doc/index.html
index e58666b..21adf06 100755
--- a/doc/index.html
+++ b/doc/index.html
@@ -324,7 +324,7 @@
           <td> 
             <hr>
             <p class="SectionBody">This project is hosted on : <br>
-              <A href="http://sourceforge.net/projects/nagios-snmp"> <IMG src="http://sourceforge.net/sflogo.php?group_id=134917&type=5" width="210" height="62" border="0" alt="SourceForge.net Logo" /></A> 
+              <A href="http://sourceforge.net/projects/nagios-snmp">http://sourceforge.net/projects/nagios-snmp</A> 
             </p>
             <p class="SectionBody"> Nagios and the Nagios logo are registered 
               trademarks of Ethan Galstad.</p>
diff --git a/doc/index_commands.html b/doc/index_commands.html
index 66bac6c..3c8f03a 100755
--- a/doc/index_commands.html
+++ b/doc/index_commands.html
@@ -109,7 +109,7 @@
 
         <tr> 
 
-          <td> <A href="http://sourceforge.net/projects/nagios-snmp"> <IMG src="http://sourceforge.net/sflogo.php?group_id=134917&amp;type=3" width="110" height="37" border="0" alt="SourceForge.net Logo" /></A> 
+          <td> <A href="http://sourceforge.net/projects/nagios-snmp">http://sourceforge.net/projects/nagios-snmp</A> 
 
           </td>
 
@@ -2014,7 +2014,7 @@
 
           <p class="SectionBody">This project is hosted on : <br>
 
-            <A href="http://sourceforge.net/projects/nagios-snmp"> <IMG src="http://sourceforge.net/sflogo.php?group_id=134917&type=5" width="210" height="62" border="0" alt="SourceForge.net Logo" /></A> 
+            <A href="http://sourceforge.net/projects/nagios-snmp">http://sourceforge.net/projects/nagios-snmp</A> 
 
           </p>
 
diff --git a/doc/snmp_cpfw.html b/doc/snmp_cpfw.html
index ac84d46..89bf324 100755
--- a/doc/snmp_cpfw.html
+++ b/doc/snmp_cpfw.html
@@ -439,7 +439,7 @@
 
             <p class="SectionBody">This project is hosted on : <br>
 
-              <A href="http://sourceforge.net/projects/nagios-snmp"> <IMG src="http://sourceforge.net/sflogo.php?group_id=134917&type=5" width="210" height="62" border="0" alt="SourceForge.net Logo" /></A> 
+              <A href="http://sourceforge.net/projects/nagios-snmp">http://sourceforge.net/projects/nagios-snmp</A> 
 
             </p>
 
diff --git a/doc/snmp_css.html b/doc/snmp_css.html
index 84fe3f4..c581d85 100755
--- a/doc/snmp_css.html
+++ b/doc/snmp_css.html
@@ -353,7 +353,7 @@ Usage: ./check_snmp_css_main.pl [-v] -H &lt;host&gt; -C &lt;snmp_community&gt; [
 
             <p class="SectionBody">This project is hosted on : <br>
 
-              <A href="http://sourceforge.net/projects/nagios-snmp"> <IMG src="http://sourceforge.net/sflogo.php?group_id=134917&type=5" width="210" height="62" border="0" alt="SourceForge.net Logo" /></A> 
+              <A href="http://sourceforge.net/projects/nagios-snmp">http://sourceforge.net/projects/nagios-snmp</A> 
 
             </p>
 
diff --git a/doc/snmp_env.html b/doc/snmp_env.html
index f847bd9..576d6a5 100755
--- a/doc/snmp_env.html
+++ b/doc/snmp_env.html
@@ -342,7 +342,7 @@
 
             <p class="SectionBody">This project is hosted on : <br>
 
-              <A href="http://sourceforge.net/projects/nagios-snmp"> <IMG src="http://sourceforge.net/sflogo.php?group_id=134917&type=5" width="210" height="62" border="0" alt="SourceForge.net Logo" /></A> 
+              <A href="http://sourceforge.net/projects/nagios-snmp">http://sourceforge.net/projects/nagios-snmp</A> 
 
             </p>
 
diff --git a/doc/snmp_int.html b/doc/snmp_int.html
index aac15fc..6284623 100755
--- a/doc/snmp_int.html
+++ b/doc/snmp_int.html
@@ -660,7 +660,7 @@
 
             <p class="SectionBody">This project is hosted on : <br>
 
-              <A href="http://sourceforge.net/projects/nagios-snmp"> <IMG src="http://sourceforge.net/sflogo.php?group_id=134917&type=5" width="210" height="62" border="0" alt="SourceForge.net Logo" /></A> 
+              <A href="http://sourceforge.net/projects/nagios-snmp">http://sourceforge.net/projects/nagios-snmp</A> 
 
             </p>
 
diff --git a/doc/snmp_load.html b/doc/snmp_load.html
index 0c48eb8..78697db 100755
--- a/doc/snmp_load.html
+++ b/doc/snmp_load.html
@@ -379,7 +379,7 @@ SNMP Load &amp; CPU Monitor for Nagios version 1.3<br>
 
             <p class="SectionBody">This project is hosted on : <br>
 
-              <A href="http://sourceforge.net/projects/nagios-snmp"> <IMG src="http://sourceforge.net/sflogo.php?group_id=134917&type=5" width="210" height="62" border="0" alt="SourceForge.net Logo" /></A> 
+              <A href="http://sourceforge.net/projects/nagios-snmp">http://sourceforge.net/projects/nagios-snmp</A> 
 
             </p>
 
diff --git a/doc/snmp_mem.html b/doc/snmp_mem.html
index 1a51886..c93c185 100755
--- a/doc/snmp_mem.html
+++ b/doc/snmp_mem.html
@@ -457,7 +457,7 @@ Usage: ./check_snmp_mem.pl [-v] -H &lt;host&gt; -C &lt;snmp_community&gt; [-2] |
 
             <p class="SectionBody">This project is hosted on : <br>
 
-              <A href="http://sourceforge.net/projects/nagios-snmp"> <IMG src="http://sourceforge.net/sflogo.php?group_id=134917&type=5" width="210" height="62" border="0" alt="SourceForge.net Logo" /></A> 
+              <A href="http://sourceforge.net/projects/nagios-snmp">http://sourceforge.net/projects/nagios-snmp</A> 
 
             </p>
 
diff --git a/doc/snmp_process.html b/doc/snmp_process.html
index 50c3735..4af115a 100755
--- a/doc/snmp_process.html
+++ b/doc/snmp_process.html
@@ -332,7 +332,7 @@
         <tr> 
           <td> 
             <p class="SectionBody">This project is hosted on : <br>
-              <A href="http://sourceforge.net/projects/nagios-snmp"> <IMG src="http://sourceforge.net/sflogo.php?group_id=134917&type=5" width="210" height="62" border="0" alt="SourceForge.net Logo" /></A> 
+              <A href="http://sourceforge.net/projects/nagios-snmp">http://sourceforge.net/projects/nagios-snmp</A> 
             </p>
             <p class="SectionBody"> Nagios and the Nagios logo are registered 
               trademarks of Ethan Galstad.</p>
diff --git a/doc/snmp_storage.html b/doc/snmp_storage.html
index 98bc5e1..14488bf 100755
--- a/doc/snmp_storage.html
+++ b/doc/snmp_storage.html
@@ -1285,7 +1285,7 @@
 
 
 
-              <A href="http://sourceforge.net/projects/nagios-snmp"> <IMG src="http://sourceforge.net/sflogo.php?group_id=134917&type=5" width="210" height="62" border="0" alt="SourceForge.net Logo" /></A> 
+              <A href="http://sourceforge.net/projects/nagios-snmp">http://sourceforge.net/projects/nagios-snmp</A> 
 
 
 
diff --git a/doc/snmp_vrrp.html b/doc/snmp_vrrp.html
index 66e37be..ca8300d 100755
--- a/doc/snmp_vrrp.html
+++ b/doc/snmp_vrrp.html
@@ -324,7 +324,7 @@
 
             <p class="SectionBody">This project is hosted on : <br>
 
-              <A href="http://sourceforge.net/projects/nagios-snmp"> <IMG src="http://sourceforge.net/sflogo.php?group_id=134917&type=5" width="210" height="62" border="0" alt="SourceForge.net Logo" /></A> 
+              <A href="http://sourceforge.net/projects/nagios-snmp">http://sourceforge.net/projects/nagios-snmp</A> 
 
             </p>
 
diff --git a/doc/snmp_windows.html b/doc/snmp_windows.html
index 4a8f513..2ae468c 100755
--- a/doc/snmp_windows.html
+++ b/doc/snmp_windows.html
@@ -243,7 +243,7 @@
         <tr> 
           <td> 
             <p class="SectionBody">This project is hosted on : <br>
-              <A href="http://sourceforge.net/projects/nagios-snmp"> <IMG src="http://sourceforge.net/sflogo.php?group_id=134917&type=5" width="210" height="62" border="0" alt="SourceForge.net Logo" /></A> 
+              <A href="http://sourceforge.net/projects/nagios-snmp">http://sourceforge.net/projects/nagios-snmp</A> 
             </p>
             <p class="SectionBody"> Nagios and the Nagios logo are registered 
               trademarks of Ethan Galstad.</p>
